src/main/java/com/rocketden/main/model/Room.java

```diff
@@ -47,6 +47,8 @@
     @Setter(AccessLevel.PRIVATE)
     private List<User> users = new ArrayList<>();
 
+    private Boolean isActive;
```

Comment:
Maybe set a default value of false? That way we don't have to deal with null values